planetiler-core/src/main/java/com/onthegomap/planetiler/archive/TileArchives.java

```diff
@@ -0,0 +1,75 @@
+package com.onthegomap.planetiler.archive;
```

Comment:
Not sure if `TileArchives` utilities for working with tile archives makes the most sense, or collapse `TileArchives` and `TileArchiveMetadata` into one `record TileArchive` that also has static factory methods?

Reply:
oops sorry I meant collapse `TileArchives` and `TileArchiveConfig`